src/Frontend/Page/Overview.hs

```diff
@@ -175,6 +175,7 @@ instance ToHtml Tabs where
 -- * idea lists
 
 data ListItemIdeaContext  -- TODO: rename to 'ListLocation'?  'Context' is used for too many things.
+                          -- 'Position'?  i think my favorite so far would be `WhatList`.  ~~mf
```

Comment:
This type is only for pass the informatin where the ListItemIdea values are generated, we have only one function which converts the `ListItemIdea` value to `Html`, but the rendering depend on the caller, so it is context dependentent, that's why it is called `ListItemIdeaContext`. If we interpret location as a location where the idea is listed that's ok, but it has nothing to do with the position of the idea in the list of ideas.